packages/plugin-doc-coverage/mocks/source-files.mock.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  ClassDeclaration,
+  EnumDeclaration,
+  FunctionDeclaration,
+  InterfaceDeclaration,
+  SourceFile,
+  SyntaxKind,
+  TypeAliasDeclaration,
+  VariableStatement,
+} from 'ts-morph';
+import type { CoverageType } from '../src/lib/runner/models';
+
+export function sourceFileMock(
```

Comment:
See https://github.com/code-pushup/cli/pull/896/files#r1894665959
lets reorganise the mocks

Reply:
But this file is like a generator of mocks, maybe it would be cool inside mocks/source-files.generator.ts or something like that

Reply:
If it is used let’s keep it if it’s motivated I see no point of keeping it